site/content/docs/5.2/components/carousel.md

```diff
@@ -236,8 +236,12 @@ Carousels support swiping left/right on touchscreen devices to move between slid
 
 ## Dark variant
 
+{{< deprecated-in "5.3.0" >}}
+
 Add `.carousel-dark` to the `.carousel` for darker controls, indicators, and captions. Controls have been inverted from their default white fill with the `filter` CSS property. Captions and controls have additional Sass variables that customize the `color` and `background-color`.
 
+{{< callout-dark-variants "carousel" >}}
```

Comment:
Reading this change, one would understand it better if the callout name contained the notion of deprecation: e.g. "callout-deprecated-dark-variants".